index.html

```diff
@@ -1,9 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <script defer src="https://use.fontawesome.com/releases/v5.0.7/js/all.js"></script>
```

Comment:
Put the JS code in Index.js

Reply:
ye aayan yuvi ne hi daala tha apni coding me i havent done anything with it

Reply:
this js code was already there in the coding done by @yuvraajnarula

Reply:
Yes now put it in index.js